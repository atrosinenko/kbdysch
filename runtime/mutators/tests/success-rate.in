[ENV:POPULATE_SUCCESS_RATE=1]
[BEGIN:Dummy success rate]
[END:After dummy success rate]
Input
