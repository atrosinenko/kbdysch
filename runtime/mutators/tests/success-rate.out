
Usage count (queue)             Usage count (all)             Success (queue)               Success (all)                 Success (queue-to-all)

half_2 (2.00)                   half_2 (2.00)                 rare_3 (0.33)                 rare_3 (0.33)                 frequent_4 (1.00)
rare_3 (3.00)                   rare_3 (3.00)                 half_2 (0.50)                 half_2 (0.50)                 rare_3 (1.00)
frequent_4 (4.00)               frequent_4 (4.00)             frequent_4 (0.75)             frequent_4 (0.75)             half_2 (1.00)
Never executed: never_0 (0)
Always failing: never_2 (2)
Always succeding: always_3 (3)
